src/main/controller/UserController.ts

```diff
@@ -123,10 +123,12 @@ export class UserController {
   // (9 - CC & WP) Retrieve detailed information about workout (title, duration, video link), exercises and their order, and info regarding exercises
   getDetailedWorkout = async (req: Request, res: Response): Promise<any> => {
     try {
+      console.log(db);
       const workout = await db.User.findOne({
         where: {
           id: req.params.userId,
         },
+        attributes: [],
```

Comment:
Can we remove this or does it complain?

Reply:
I don't need any attributes from the `User` db. So having an empty array for attributes guarantees that. I can remove it though

Reply:
It doesn't really matter in the response as I'm not sending any information about the user (I'm responding with `workout.workouts` instead of `workout`).

But, this is how it looks like if I sent the actual query response and had `attributes: []` (i.e. responding with `workout` instead of `workout.workouts`):
```json
{
    "workouts": [
        {
            "name": "15 Minutes to Intense Abs",
            "totalWorkoutTime": "510",
            "imageUrl": "thisissampleimageurl2",
            "Exercises": [ /* populated array of exercises */]
        }
    ]
}
```

This is what it would look like without `attributes: []`: 
```json
{
    "id": "user_uuid",
    "firstName": "user_first_name",
    "lastName": "user_last_name",
    "username": "user_username",
    "email": "user_email",
    "createdAt": "2021-11-23T21:02:51.023Z",
    "updatedAt": "2021-11-23T21:02:51.023Z",
    "workouts": [
        {
            "name": "15 Minutes to Intense Abs",
            "totalWorkoutTime": "510",
            "imageUrl": "thisissampleimageurl2",
            "Exercises": [ /* populated array of exercises */]
        }
    ]
}
```

So essentially, having the `attributes: []` removed the user table columns as property fields from the query.

Reply:
Looks good. Small note: do you want to return `workout.workouts[0]` instead of `workout.workouts` then? It seems you're returning an array, whereas the result of getting one workout probably should be an (workout) object, instead of an array.

Reply:
I'll return `workout.workouts[0]`